server/jwt.go

```diff
@@ -57,9 +62,9 @@ func ReadOperatorJWT(jwtfile string) (*jwt.OperatorClaims, error) {
 	}
 	opc, err := jwt.DecodeOperatorClaims(claim)
 	if err != nil {
-		return nil, err
+		return "", nil, err
 	}
-	return opc, nil
+	return claim, opc, nil
```

Comment:
For readability (at the expense of more changes that are not really changes), I would have renamed the "claim" variable to "jwt" if this is what it is. The "opc" variable seem to be the operator claim.

Reply:
fixed